format-message.js

```diff
@@ -272,3 +272,46 @@ https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-report
 📌 Escriba 6 para noticias y prensa
 📌 Escriba 0 para volver al menú
 `;
+
+function formatNewsMessage(newsList, whoNumber) {
+  let msg = `*LATEST NEWS & PRESS*
+
+`;
+  if (newsList.items.length > 5) {
+    newsList.items.length = 5;
+  }
+  newsList.items.forEach(item => {
+    let description = item.contentSnippet;
+    if (description.length > 300) {
+      description = description.substring(0,description.indexOf(".", 100)+1)
+    }
+    msg += `• *${item.title}*
+${description}
+Read more here: ${item.link}
```

Comment:
@KaitCrawford Look 👍  🍰  to me. 
The last line on News report description here seems to be like this on the bot `Find the full Situation Report #140 here: https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200608-covid-19-sitrep-140.pdf?sfvrsn=2f310900_2`

Reply:
Yup. I'm not adding that since there isn't an easy way to always pull the latest one.